task_11/index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <!--    Define keywords for search engines:-->
```

Comment:
Code must be self-documented as much as possible.

Highly suggest you avoid stupid comments for lines which are clear without it.